object-construction-checker/tests/socket/ZookeeperReport6.java

```diff
@@ -1,13 +1,13 @@
 // Based on a Zookeeper false positive that requires unconnected socket support.
 
-// @skip-test
-
 import java.nio.channels.SocketChannel;
 import java.io.IOException;
 
 class ZookeeperReport6 {
     SocketChannel createSock() throws IOException {
         SocketChannel sock;
+        // false positive
```

Comment:
why change skipping the test to a comment about a false positive?

Reply:
because I was fixing it, see my latest commit